src/net.h

```diff
@@ -533,6 +537,7 @@ class CConnman
     NetEventsInterface* m_msgproc;
     /** Pointer to this node's banman. May be nullptr - check existence before dereferencing. */
     BanMan* m_banman;
+    std::vector<CAddress> m_anchors;
```

Comment:
I think it'd be good to add a doxygen comment to this new member variable. This vector stores addresses that we should attempt to make block-relay-only connections to, not connections that we'll necessarily store as anchor connections next time we shut down.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-693330527).

Reply:
Thanks for adding this. Sorry I wasn't clearer, but I didn't mean you should add "not connections that we'll necessarily store as anchor connections next time we shut down" as a comment. That was me explaining to you _why_ a comment might be helpful. In general, I don't think comments should explain what the code_isn't_ doing.

If you have to touch this PR again, I'd suggest "Addresses that were saved during the previous shutdown. We'll attempt to make block-relay-only connections to them."

Reply:
[Improved](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-693556607).